### ISSUE TITLE
skipNonDigests runtime error in 2.12.2

### ISSUE THREAD

**Issue by varju** [bertramdev/asset-pipeline#143]
After upgrading to 2.12.2, I am seeing an exception in certain cases while running assetCompile.

I don't currently have a simple test case to demonstrate this problem, as this was triggered from within a large multiproject build.  That said, the exception looks very straightforward, and points to a new line that was added in 2.12.2:

```
Caused by: groovy.lang.MissingPropertyException: No such property: skipNonDigests for class: asset.pipeline.AssetCompiler
	at asset.pipeline.AssetCompiler$_removeDeletedFiles_closure9.doCall(AssetCompiler.groovy:397)
	at asset.pipeline.AssetCompiler.removeDeletedFiles(AssetCompiler.groovy:369)
	at asset.pipeline.AssetCompiler.this$2$removeDeletedFiles(AssetCompiler.groovy)
	at asset.pipeline.AssetCompiler.compile(AssetCompiler.groovy:123)
	at asset.pipeline.AssetCompiler$compile.call(Unknown Source)
	at asset.pipeline.gradle.AssetCompile.compile(AssetCompile.groovy:221)
```

It looks like https://github.com/bertramdev/asset-pipeline/commit/5c0d2b5566358d1fc87521a7289c324cceb569df added references to a missing `skipNonDigests` variable.  Perhaps this should have been `options.skipNonDigests`?

**Comment by davydotcom** [comment]
oops, my bad fixed in 2.12.3

**Comment by varju** [comment]
Looks good now, thank you!
